README.md

```diff
@@ -79,14 +79,53 @@ This will scaffold a basic workspace inside the `packages` folder.
 
 ### Publish a package
 
--   Run `pnpm build` to build the packages.
+The process relies 100% on the github actions workflow. PR are merged into main, which triggers a
```

Comment:
I would add a mention that every PR needs a changeset

Reply:
@f-elix Good call, see 5d27a91dd3